### PR TITLE
Update code analysis

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

**Comment by hvelarde** [comment]
@tcurvelo I need your help here also; please double check and test manually if possible.

**Comment by tcurvelo** [comment]
Is this it?
I think all tests passed.

File: buildout.cfg [hunk 1]
```diff
@@ -42,6 +36,5 @@ recipe = collective.recipe.omelette
 eggs = ${instance:eggs}
 
 [versions]
-# use latest version of coverage and setuptools
+# use latest version of coverage
 coverage =
-setuptools =
```
Comment (tcurvelo) [inline buildout.cfg]: Why remove this?

Comment (hvelarde) [inline buildout.cfg]: because is no longer needed

**Pull request merged**